tests/t5_cp.py

```diff
@@ -59,3 +59,62 @@ def tst_cp(self):
 
         finally:
             shutil.rmtree(tempdir)
+
+
+    def test_cp_inode_invalidate(self):
+        if os.getuid() != 0:
+            pytest.skip('test_cp_inode_invalidate requires root, skipping.')
+
+        self.passphrase = None
+        self.mkfs()
+
+        # Create monkeypatched mount.s3ql
+        # Override pyfuse3.invalidate_inode : Drop kernel dentries and inodes cache
+        # just before calling pyfuse3.invalidate_inode
+        patched_mount = """
```

Comment:
Please put this in a separate file, e.g. `tests/t5_cp_mount_helper.py`

Reply:
:+1: